datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1930,16 +1944,20 @@ impl LogicalPlan {
                             )
                         }
                     },
-                    LogicalPlan::Limit(Limit {
-                        ref skip,
-                        ref fetch,
-                        ..
-                    }) => {
+                    LogicalPlan::Limit(limit) => {
+                        // Attempt to display `skip` and `fetch` as literals if possible, otherwise as expressions.
```

Comment:
Display literals as before to avoid breaking too many tests. Maybe we could display them in the expr-style through a follow-up PR. For example, `1` -> `Int64(1)`.